### DISCUSSION TITLE
Just FYI early feedback

### DISCUSSION THREAD

**Discussion by LLay** [mui/toolpad#3352] [Discussions]
I might be totally missing the point of this tool (I've spent 20 minutes clicking around the docs and playing with it locally), but my primary use case would be generating React code. This would be a bridge between my designs I've made with MUI design toolkit and my actual app. Just FYI
I'm a founder working on prototyping various web apps. I've bought MUI commercial for the datagrid and love it.

**Comment by prakhargupta1** [comment]
Hi @LLay 
Toolpad Studio is for building admin dashboards. These are design-light and data-heavy apps that can be created quickly using the drag-drop UI. 
In addition to Studio, we are working on a new idea: https://github.com/mui/mui-toolpad/discussions/3311, it is a code-centric approach to the same problem of building admin apps fast. We have some prototypes to give you an idea. Let us know what you think.
